Style multi line (#58)
CosmoStat/Shear-and-PSF-Reading-Group: shearbook/contrib/style.md

```{warning}
In progress!
```
### Rendering multi-line LaTeX equations

> Although it is not necessary on classical notebooks, jupyter books require the multi-line LaTeX equations to be surrounded by $$

Reviewer: ```suggestion
Although it is not necessary in classical notebooks, Jupyter Book requires multi-line LaTeX equations to be surrounded by `$$`
```

```

> ### Rendering multi-line euqations

Reviewer 2: ```suggestion
### Rendering multi-line LaTeX equations
```
